packages/grafana-ui/src/types/time.ts

```diff
@@ -21,26 +21,15 @@ export interface IntervalValues {
   intervalMs: number;
 }
 
-export interface TimeZone {
-  raw: string;
-  isUtc: boolean;
-}
-
-export const parseTimeZone = (raw: string): TimeZone => {
-  return {
-    raw,
-    isUtc: raw === 'utc',
-  };
-};
+export type TimeZone = 'utc' | 'browser';
```

Comment:
Can we change this to an enum instead to get safe typings instead of using "magic" strings all over like `'utc'` and `'browser'`?

Reply:
This is type safe, any other value will give you error

Reply:
Yes, but a bit easier to find usage of an enum than strings. So instead of doing `timeZone === 'utc' you would do `timeZone === TimeZone.UTC`.

Reply:
true, just this will be huge enum in that case, to support all timezones names and UTC offset variants. maybe constants instead? but that will not be clear that you should use a constant.

Reply:
maybe this 

```
export type TimeZoneUtc = 'utc';
export type TimeZoneBrowser = 'browser';
export type TimeZone = TimeZoneBrowser | TimeZoneUtc | string;
```

but there is nothing enforcing use of TimeZoneBrowser etc

Reply:
This is no biggie we can use the current solution for now, just nitpick I guess. 

> true, just this will be huge enum in that case, to support all timezones names and UTC offset variants.

Using `type TimeZone = 'utc' | 'browser';` will also be a huge list! 

Probably easier to handle later when implementing support for more time zones. As you discuss we may want to use some sort of `TimeZoneOffset` field/type later which describes the offset in minutes compared to UTC? 

Seems like there are [38 offsets](https://en.wikipedia.org/wiki/List_of_UTC_time_offsets).